charts/shoot-core/components/charts/network-policies/templates/allow-from-to-nginx.yaml

```diff
@@ -0,0 +1,25 @@
+kind: NetworkPolicy
+apiVersion: networking.k8s.io/v1
+metadata:
+  annotations:
+    gardener.cloud/description: |
+      Allows Egress and Ingress from the nginx-controller to all traffice
+  name: gardener.cloud--allow-to-from-nginx
+  namespace: kube-system
+  labels:
+    origin: gardener
+spec:
+  podSelector:
+    matchLabels:
+      app: nginx-ingress
+  policyTypes:
+  - Ingress
+  - Egress
+  egress:
+  - to:
+    - ipBlock:
+        cidr: 0.0.0.0/0
+  ingress:
```

Comment:
Allow ingress from all Pods and from all namespaces. For example if Pod-A a tries to access a service via its LoadBalancer IP, if there is no policy allowing that Pod-A is now allowed at the ingress, it won't work.

Reply:
```yaml
egress:
- {}
```
should be sufficient https://kubernetes.io/docs/concepts/services-networking/network-policies/#default-deny-all-ingress-and-all-egress-traffic